test/deck_test.rb

```diff
@@ -11,53 +11,58 @@ def setup
     @card3 = Card.new(:heart, 'Ace', 14)
     @card4 = Card.new(:club, '5', 5)
     @deck = Deck.new(@cards)
+    @deck.add_card(@card1)
+    @deck.add_card(@card2)
+    @deck.add_card(@card3)
   end
 
   def test_it_exists
     assert_instance_of Deck, @deck
   end
 
   def test_it_has_no_cards
-    assert_equal [], @deck.cards
+    deck = Deck.new(@cards)
```

Comment:
this one test failed when I updated the setup method. Now it doesn't, because I made a different `deck` object.